blocks/resizer-image-block/getResizedImageData.test.js

```diff
@@ -197,13 +197,13 @@ describe('get resized image data helper on the server-side', () => {
 
     // if the same resizer key is used this won't change
     expect(filterValues).toEqual([
-      '/JH9YE5rXU3WoFv5vs342UvuC1gA=filters:format(jpg):quality(70)/',
-      '/oVCzXp5WcaNYOq9omFEyWoMCo78=filters:format(jpg):quality(70)/',
-      '/31mmYPove0AN7aFn_vwKJ0qhZeI=filters:format(jpg):quality(70)/',
-      '/TkU_4vX7gHlX4WBLLwCDMrLuQj4=filters:format(jpg):quality(70)/',
+      'JH9YE5rXU3WoFv5vs342UvuC1gA=filters:f=jpg:q=70/',
```

Comment:
wait, did we want to default to the jpg format? so as not to trigger the googlebot jpg check?

Reply:
There is not other format supported at this point in blocks/resizer-image-block/index.js